tasks/src/main/scala/dagr/tasks/picard/GatherBamFiles.scala

```diff
@@ -25,12 +25,14 @@
 
 package dagr.tasks.picard
 
+import dagr.core.tasksystem.PipeOut
 import dagr.tasks.DagrDef._
+import dagr.tasks.DataTypes.SamOrBam
 
 import scala.collection.mutable.ListBuffer
 
 class GatherBamFiles(val in: Seq[PathToBam], val out: PathToBam, createIndex: Option[Boolean] = Some(true))
-  extends PicardTask(createIndex=createIndex) {
+  extends PicardTask(createIndex=createIndex) with PipeOut[SamOrBam] {
```

Comment:
Seems fine .... but I see you sneaking this in in a second commit in an unrelated PR 😛